python/mujinplanningclient/realtimerobotplanningclient.py

```diff
@@ -1,57 +1,67 @@
 # -*- coding: utf-8 -*-
 # Copyright (C) 2012-2015 MUJIN Inc
 
+# mujin imports
 from . import json
 from . import planningclient
 
+# logging
 import logging
 log = logging.getLogger(__name__)
 
+
 class RealtimeRobotPlanningClient(planningclient.PlanningClient):
-    """Mujin controller client for realtimerobot task"""
+    """Mujin planning client for the RealtimeRobot task"""
+
     _robotname = None  # Optional name of the robot selected
     _robotspeed = None  # Speed of the robot, e.g. 0.4
     _robotaccelmult = None  # Current robot accel mult
     _envclearance = None  # Environment clearance in millimeters, e.g. 20
     _robotBridgeConnectionInfo = None  # dict holding the connection info for the robot bridge.
-    
+
+    _deprecated = None # used to mark arguments as deprecated (set argument default value to this)
+
     def __init__(self, robotname='', robotspeed=None, robotaccelmult=None, envclearance=10.0, robotBridgeConnectionInfo=None, **kwargs):
         """
         Args:
-            controllerip (str, optional): Ip or hostname of the mujin controller, e.g. controller14 or 172.17.0.2
-            controllerurl (str, optional): (Deprecated; use controllerip instead) URL of the mujin controller, e.g. http://controller14.
-            controllerusername (str): Username for the Mujin controller, e.g. testuser
-            controllerpassword (str): Password for the Mujin controller
             robotname (str, optional): Name of the robot, e.g. VP-5243I
-            scenepk (str, optional): Primary key (pk) of the scene, e.g. irex_demo.mujin.dae
             robotspeed (float, optional): Speed of the robot, e.g. 0.4
             robotaccelmult (float, optional): Optional multiplier for the robot acceleration.
-            tasktype (str, optional): Type of the task, e.g. 'binpicking', 'handeyecalibration', 'itlrealtimeplanning3'
             envclearance (str, optional): Environment clearance in millimeter, e.g. 20
             robotBridgeConnectionInfo (str, optional): dict holding the connection info for the robot bridge.
-            taskzmqport (int, optional): Port of the task's ZMQ server, e.g. 7110
-            taskheartbeatport (int, optional): Port of the task's ZMQ server's heartbeat publisher, e.g. 7111
+            taskzmqport (int, optional): Port of the task's ZMQ server. Default: 11000
```

Comment:
You mean that "e.g. 7110" part dissapeared? Looks like reasonable change since we state Default (its same defaults in the base class as well). But I dont mind to return it. Done.